src/dssgsolve/settings.py

```diff
@@ -147,14 +166,15 @@
     EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
 else:
     EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+
 EMAIL_FILE_PATH = '/tmp/app-messages'
-EMAIL_HOST = config('EMAIL_HOST')
-EMAIL_PORT = config('EMAIL_PORT', cast=int)
-EMAIL_HOST_USER = config('EMAIL_HOST_USER')
-EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
-EMAIL_USE_TLS = config('EMAIL_USE_TLS', default=True, cast=bool)
+# EMAIL_HOST = config('EMAIL_HOST')
+# EMAIL_PORT = config('EMAIL_PORT', default=587, cast=int)
+# EMAIL_HOST_USER = config('EMAIL_HOST_USER')
+# EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
+# EMAIL_USE_TLS = config('EMAIL_USE_TLS', default=True, cast=bool)
 
-DEFAULT_FROM_EMAIL = config('EMAIL_FROM_ADDRESS')
+# DEFAULT_FROM_EMAIL = config('EMAIL_FROM_ADDRESS')
```

Comment:
As mentioned above, it did not appear that these settings are in fact yet in use; and, having to dummy them up in the outside configuration seemed unwise in production and an unnecessary pain.

Reply:
Just to make sure we are on the same page, the site currently uses them when setting DEBUG=False, as it uses the SMTP backend for sending email (when in debug mode, it sends the emails to the filesystem). So when you say the settings are not in use, do you mean in this specific demo deployment?

Reply:
Yes, I think, in the frenzy of development, you're discussing _code paths_, but which are not in use, because, (unless *I'm* missing something huge), **there is no SMTP server**. Moreover, once this is set up, the `EMAIL_FROM_ADDRESS` will not be `info@dssg.com` :smile: 

I commented this out because it was a barrier to demo deployment.

With email sending set up, (which, BTW, means proving to AWS that we aren't spammers, beyond simply authenticating against their SMTP relay), then we can configure all this. That's why I left it, simply commented out.

As I was saying, at that point, to avoid confusion and unnecessary dummy values, we might want to say something like:

    if DEBUG:
        EMAIL_BACKEND = ... # not smtp
        EMAIL_FILE_PATH = ...
    else:
        EMAIL_BACKEND = ... # smtp
        DEFAULT_FROM_EMAIL = config('EMAIL_FROM_ADDRESS')  # this value is REQUIRED
        ...